gbus/builder/builder.go

```diff
@@ -79,7 +79,8 @@ func (builder *defaultBuilder) Build(svcName string) gbus.Bus {
 		providerLogger := gb.Log().WithField("provider", "mysql")
 		mysqltx, err := mysql.NewTxProvider(builder.txConnStr)
 		if err != nil {
-			panic(err)
+			logerr := fmt.Errorf("grabbit faild on mysql.NewTxProvider. builder.txConnStr: %v, builder.connStr: %v, error: %v", builder.txConnStr, builder.connStr, err)
```

Comment:
@kerenbe4 please remove the connection string from the error message as it may contain user credentials that will later get logged when the application will log the error.
In addition:

- logger is probably not the right variable name for an error :)
- "grabbit faild on mysql.NewTxProvider" contains a typo and should probably be more descriptive  as to the nature of the error. I would prefer "transaction provider failed  creating a transaction"

Reply:
In addition, grabbit already integrates with [https://github.com/emperror/errors](https://github.com/emperror/errors)which has a nice error wrapping feature that is can make the code above better.